Doc/c-api/init.rst

```diff
@@ -1083,6 +1083,31 @@ All of the following functions must be called after :c:func:`Py_Initialize`.
 
    .. versionadded:: 3.8
 
+.. c:type:: PyObject* (*PyFrameEvalFunction)(PyThreadState *tstate, PyFrameObject *frame, int throwflag)
```

Comment:
It may be nice to say that from 3.6.x  to 3.8.0 (inclusive) the frame eval function did not have the `tstate` parameter (although the api was added on 3.8.1 previous uses which already set the frame eval only expected the frame and throwflag) -- I know this will break the debugger from 3.8.0 to 3.8.1 and there may be other users in the wild...

Reply:
Sure, done.